src/Builder/ContentBuilder.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Mollie\Builder;
+
+class ContentBuilder implements \Mollie\Builder\TemplateBuilderInterface
+{
+	/**
+	 * @var array
+	 */
+	private $templateBlocks = [];
+
+	public function addTemplateBlock(TemplateBuilderInterface $templateBuilder)
+	{
+		$this->templateBlocks = array_merge($this->templateBlocks, $templateBuilder->buildParams());
+
+		return $this;
+	}
```

Comment:
can we not violate interface and not expose yet another public method ? 

This service can be used without DY container


```php
(new ContentBuilder(TemplateBuilderInterface $templateBuilder))->buildParams()
```